README.md

```diff
@@ -734,7 +734,7 @@ What is going on here?
     2
     >>> dictionary in another_set
     True
-    >>> another_set.add(another_ordered_dict)
+    >>> another_set.add(dictionary)
```

Comment:
Actually, the [statement above it ](https://github.com/satwikkansal/wtfpython/pull/244/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R732)should be `dictionary`. Nice catch though, thanks..